File: app/src/test/java/com/example/mike/mp3player/client/utils/IntentUtilsTest.java
package com.example.mike.mp3player.client.utils;

import android.content.Context;
import android.content.Intent;

import androidx.test.platform.app.InstrumentationRegistry;

import org.junit.Test;
import org.junit.runner.RunWith;
import org.robolectric.RobolectricTestRunner;

import static org.junit.Assert.assertNotNull;


@RunWith(RobolectricTestRunner.class)
public class IntentUtilsTest {

    @Test
    public void testCreateGoToMediaPlayerActivity() {
        Context context = InstrumentationRegistry.getInstrumentation().getContext();
        Intent result = IntentUtils.createGoToMediaPlayerActivity(context);
        assertNotNull(result);
    }
}

File: client/src/test/java/com/github/goldy1992/mp3player/client/utils/IntentUtilsTest.kt
package com.github.goldy1992.mp3player.client.utils

import androidx.test.platform.app.InstrumentationRegistry
import com.github.goldy1992.mp3player.client.utils.IntentUtils.createGoToMediaPlayerActivity
import org.junit.Assert
import org.junit.Test
import org.junit.runner.RunWith
import org.robolectric.RobolectricTestRunner

@RunWith(RobolectricTestRunner::class)
class IntentUtilsTest {
    @Test
    fun testCreateGoToMediaPlayerActivity() {
        val context = InstrumentationRegistry.getInstrumentation().context
        val result = createGoToMediaPlayerActivity(context)
        Assert.assertNotNull(result)
    }
}